server/models/user.model.js

```diff
@@ -42,7 +42,12 @@ module.exports = (sequelize, DataTypes) => {
         email: {
             type: DataTypes.STRING,
             unique: true,
-            allowNull: false,
+            allowNull: true,
```

Comment:
Does Postgres allow any sort of OR enforcement on columns? I suspect the answer is no but it would be useful to enforce.

Reply:
Postgres allows it by doing a CHECK, https://stackoverflow.com/questions/5447880/adding-an-one-out-of-two-not-null-constraint-in-postgresql , But there is no way to make this appear the the table with sequalize the way were are generating the tables currently. We would have to have add a create script and stop syncing the user table.